src/test_workflow/perf_test/perf_test_suite.py

```diff
@@ -23,16 +22,20 @@ def __init__(self, bundle_manifest, endpoint, security, current_workspace, args)
             f" --warmup-iters {self.args.warmup_iters} --test-iters {self.args.test_iters}"
         )
 
+        if test_results_path is not None:
+            self.command = (
+                f"pipenv run python test_config.py -i {self.endpoint} -b {self.manifest.build.id}"
```

Comment:
This is Python code calling python code? It should import the code and call it directly.

Reply:
The code is in a separate package which is not yet open to the public. We pull down the code from the repo and execute it.
Do you mean using `importlib` or something similar for starting the execution?